react-native-image-editor.podspec

```diff
@@ -13,8 +14,30 @@ Pod::Spec.new do |s|
   s.platform     = :ios, "9.0"
 
   s.source       = { :git => "https://github.com/callstack/react-native-image-editor.git", :tag => "#{s.version}" }
-  s.source_files  = "ios/**/*.{h,m}"
+  s.source_files  = "ios/**/*.{h,m,mm}"
 
   s.dependency 'React'
+  s.dependency "React-Core"
   s.dependency 'React-RCTImage'
+
+  # Use install_modules_dependencies helper to install the dependencies if React Native version >=0.71.0.
+  # See https://github.com/facebook/react-native/blob/febf6b7f33fdb4904669f99d795eba4c0f95d7bf/scripts/cocoapods/new_architecture.rb#L79.
+  if respond_to?(:install_modules_dependencies, true)
+    install_modules_dependencies(s)
+  end
+
+  # Don't install the dependencies when we run `pod install` in the old architecture.
+  if ENV['RCT_NEW_ARCH_ENABLED'] == '1' then
```

Comment:
Will it be run even if above 0.71 helper is available? I think it shouldn't - this if statement should only run for RN <= 0.70

Reply:
this is an output of `create-react-native-library@latest` template
that I used for adding new arch support 

(you can open an issue if it a bug https://github.com/callstack/react-native-builder-bob/issues)

Reply:
This is the output from Bob:

```
# Use install_modules_dependencies helper to install the dependencies if React Native version >=0.71.0.
# See https://github.com/facebook/react-native/blob/febf6b7f33fdb4904669f99d795eba4c0f95d7bf/scripts/cocoapods/new_architecture.rb#L79.
if respond_to?(:install_modules_dependencies, true)
    install_modules_dependencies(s)
else
    s.dependency "React-Core"

    # Don't install the dependencies when we run `pod install` in the old architecture.
    if ENV['RCT_NEW_ARCH_ENABLED'] == '1' then
        s.compiler_flags = folly_compiler_flags + " -DRCT_NEW_ARCH_ENABLED=1"
        s.pod_target_xcconfig    = {
            "HEADER_SEARCH_PATHS" => "\"$(PODS_ROOT)/boost\"",
            "OTHER_CPLUSPLUSFLAGS" => "-DFOLLY_NO_CONFIG -DFOLLY_MOBILE=1 -DFOLLY_USE_LIBCPP=1",
            "CLANG_CXX_LANGUAGE_STANDARD" => "c++17"
        }
<% if (project.view) { -%>
        s.dependency "React-RCTFabric"
<% } -%>
        s.dependency "React-Codegen"
        s.dependency "RCT-Folly"
        s.dependency "RCTRequired"
        s.dependency "RCTTypeSafety"
        s.dependency "ReactCommon/turbomodule/core"
    end
end
```

While here, after the first `if` there is `end` instead of `else`. Let's fix the missing `else` plus the indentation, so  it's more readable